modules/transactions.js

```diff
@@ -106,7 +106,7 @@ __private.list = function(filter, cb) {
 	if (filter.data) {
 		// PostgreSQL does not support zero byte character in utf8 strings
 		// so we have to specify text in hex representation
```

Comment:
This comment is not true anymore. it's Buffer

Reply:
@shuse2 Fixed.